VultisigApp/VultisigApp/View Models/SendCryptoViewModel.swift

```diff
@@ -157,6 +157,17 @@ class SendCryptoViewModel: ObservableObject, TransferViewModel {
                 
                 await convertToFiat(newValue: tx.amount, tx: tx)
                 
+                isLoading = false
+            }
+        case .polkadot:
+            Task {
+                await BalanceService.shared.updateBalance(for: tx.coin)
+                var gas = BigInt.zero
```

Comment:
do we need that var? 

why we're passing zero getMaxValue?

Reply:
do you mind to get it fixed , so we can get this one merged?

Reply:
Sure, sorry for the delay.